### PR TITLE
Update free-courses-hi.md 2 new course added 1. Core Java in hindi 2.…

### PR DESCRIPTION
**Pull request by nabinamallik**
… Java spring boot in hindi

I just add 2 Java Course 1. Core Java and 2. Java spring boot if you see the whole repo then my contribution is just a pinch but also I know that A Ocean also make from drops of water 😊

## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by nabinamallik** [comment]
i added some extra links

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -303,6 +303,7 @@
 
 * [Complete Java Programming in Hindi](https://www.youtube.com/playlist?list=PLmRclvVt5DtnqhXTJwd-oqVRwO3bLZCGV) - Anand Kumar, CodeitUp
 * [Core Java Programming (Hindi)](https://www.youtube.com/playlist?list=PLbGui_ZYuhij8Oplrvjt_RlDliZQgdxoV) - Rajesh Kumar, Geeky Shows
+* [Core Java in Hindi](https://youtube.com/playlist?list=PLVlQHNRLflP8kPvCvEM7ZopbRB93joJXy&si=129gUrwDvoCeajQ5) - Naresh i Technologies
```
Comment (sohansai) [inline courses/free-courses-hi.md:306]:
```suggestion
* [Core Java in Hindi](https://www.youtube.com/playlist?list=PLVlQHNRLflP8kPvCvEM7ZopbRB93joJXy) - Naresh i Technologies
```

File: courses/free-courses-hi.md [hunk 2]
```diff
@@ -313,6 +314,7 @@
 * [Java Tutorials for Beginners](https://www.youtube.com/playlist?list=PLlhM4lkb2sEhfuXL-2BDrJ67WkUdQ2v9b) - Deepak Panwar, Smart Programming
 * [Java Tutorials For Beginners In Hindi](https://www.youtube.com/playlist?list=PLu0W_9lII9agS67Uits0UnJyrYiXhDS6q) - CodeWithHarry
 * [Spring Boot Tutorial For Beginners (in Hindi)](https://www.youtube.com/playlist?list=PL5mjp3QjkuoLPS-L28yKCKyzCMX8WRVno) - ProgRank
+* [Spring Boot Tutorial In Hindi ](https://youtube.com/playlist?list=PL0zysOflRCelmjxj-g4jLr3WKraSU_e8q&si=0qL4hL44ezb-C9Cq) - Learn Code with Durgesh
```
Comment (sohansai) [inline courses/free-courses-hi.md:317]:
```suggestion
* [Spring Boot Tutorial In Hindi ](https://www.youtube.com/playlist?list=PL0zysOflRCelmjxj-g4jLr3WKraSU_e8q) - Learn Code with Durgesh
```

**Comment by eshellman** [comment]
added course in #10988

**Pull request closed without merging**